### ISSUE TITLE
Running for separate Ruby project using --project_path

### ISSUE THREAD

**Issue by tmtrademarked** [pivotal-legacy/LicenseFinder#510]
Apologies if this is well-understood already - I can't seem to find any issues referencing this, but the Known Issues of the README may be relevant here.

I'm trying to set up a centralized tool using this library to allow checks and reports for our apps. But when running this gem from a local ruby project but passing the `--project_path` pointing to another project seems to fail. This seems unexpected to me, and I'm hoping that someone could shed some light on why that is.

Is this something that you envision supporting? If not, do you have any workarounds you could suggest rather than installing `license_finder` in each project individually?

**Comment by kdykeman** [comment]
Hi @tmtrademarked, `--project_path` is something that should be supported. The known issue mentioned for Bundler in README.md is likely referencing an old issue - I'll verify and update the README if that's true.

Using the `--project_path` option for a project whose `Gemfile.lock` is in '<path-to-my-bundler-project>', the command line should look something like:
```bash
license_finder report --prepare --project-path='<path-to-my-bundler-project>'
```
If you're already doing something like that, can you fill in:
1. Which version of LicenseFinder are you running?
1. What is the command you're running? (redact anything sensitive)
1. What is the output from LicenseFinder?
1. Is any useful information yielded by running with `--debug`?

On a side note, an alternative to having a centralized installation might also be to [leverage the docker image](https://github.com/pivotal-legacy/LicenseFinder/blob/master/README.md#docker).

**Comment by tmtrademarked** [comment]
I think I got this working - it looks like installing license_finder via `gem install` made things work more cleanly, as opposed to running this from within a ruby bundle. Unfortunately, I ended up deleting all of the previous code I'd written, so I can't easily reconstruct an example to post for posterity.

Thanks, @kdykeman ! I'll take a look at using the docker image to see if that fits better in our toolchain.
